src/widgets/modals/CashTransactionModal.js

```diff
@@ -60,6 +65,25 @@ export const CashTransactionModal = ({ onConfirm }) => {
   const onChangeText = useMemo(() => text => setDescriptionBuffer(text));
   const onChangeAmount = useMemo(() => value => setAmountBuffer(value));
 
+  const renderNameLeftText = useCallback(({ firstName, lastName, name: fullName, isPatient }) =>
+    isPatient ? `${lastName}, ${firstName}` : `${fullName}`
+  );
+
+  const renderNameRightText = useCallback(({ type: nameType, dateOfBirth }) => {
+    switch (nameType) {
+      case NAME_TYPE_KEYS.FACILITY:
```

Comment:
You can also have `type === 'store'` (internal suppliers) - facility also is really terminology which isn't used very often? I think using just the names `code` would be all good, and what you have for a patient if a patient?

Reply:
f15e29f43a35879ef971b538970dcb166f17c1e5.

I think facility is generally used as a generic term for an internal or external supplier, so trying to keep consistent with that. Not sure what you mean re. patient (there's a key case for that)?

Reply:
That's then a contradiction - if a facility is an external store, then you've just added both potentially internal and external suppliers (the terms used in mobile - i.e. in `UIDatabase`) to have "facility" next to it. 

I think using `isSupplier` or `isPatient` fields would be more suitable - really, we just need to check if `isPatient` and render the extra details.

Reply:
Typo above: "external" = "internal or external" store.

Cash register needs to handle customers as well as suppliers. So I think using the `isX` is good, but will need to include `isCustomer`.

Reply:
45e11b63aaa999da1a4bb3f42c982bef8cf574c4.